app/forms/idv/api_image_upload_form.rb

```diff
@@ -88,14 +85,27 @@ def post_images_to_client
 
     def validate_pii_from_doc(client_response)
       response = Idv::DocPiiForm.new(client_response.pii_from_doc).submit
+      response = response.merge(remaining_attempts_response)
+
       track_event(
         Analytics::IDV_DOC_AUTH_SUBMITTED_PII_VALIDATION,
-        response.to_h.merge(user_id: user_uuid),
+        response.to_h,
       )
       store_pii(client_response) if client_response.success? && response.success?
 
-      # merge in the image_form_response to pick up the remaining_attempts
-      response.merge(form_response)
+      response
+    end
+
+    # A successful response providing remaining attempts and user_id to merge into other responses
+    def remaining_attempts_response
+      @remaining_attempts_response ||= Idv::DocAuthFormResponse.new(
+        success: true,
+        errors: nil,
```

Comment:
I think the build errors are from here. `errors` is converted to an a hash with `to_h`, so `nil` will throw.

```suggestion
        errors: {},
```

Reply:
Doh!

Reply:
Fixed in 0e95bb936  - also,I renamed the merged response to `extra_attributes_response` which is more accurate, and that calls `extra_attributes` which then is also used in the initial form response.